packages/webpack-cli/lib/groups/ZeroConfigGroup.js

```diff
@@ -20,18 +20,14 @@ class ZeroConfigGroup extends GroupHelper {
         if (process.env.NODE_ENV && (process.env.NODE_ENV === PRODUCTION || process.env.NODE_ENV === DEVELOPMENT)) {
             return process.env.NODE_ENV;
         } else {
-            // commander sets mode to false if --no-mode is specified
-            const noMode = this.args.mode === false;
-            if ((this.args.mode || noMode) && (this.args.dev || this.args.prod)) {
+            if (this.args.mode && (this.args.dev || this.args.prod)) {
                 logger.warn(
-                    `You provided both ${this.args.mode ? 'mode' : 'no-mode'} and ${
+                    `You provided both 'mode' and ${
                         this.args.prod ? '--prod' : '--dev'
-                    } arguments. You should provide just one. "${this.args.mode ? 'mode' : 'no-mode'}" will be used`,
+                    } arguments. You should provide just one. "mode" will be used`,
                 );
                 if (this.args.mode) {
```

Comment:
small nitpick, we don't need this if statement here anymore

Reply:
@Loonride Done :+1: